### DISCUSSION TITLE
MD Checkbox in PDF

### DISCUSSION THREAD

**Discussion by e2045** [simonhaenisch/md-to-pdf#116] [Q&A] [answered]
Hi, 

In my MD file there are some check marks. 

After converting all check marks disappear, they are converted into text like ": white_check_mark:" or ": x:" (no blank after :, i just made a blank to show you how it is).  

How could I handle it to convert checkbox the right way?

Thank you in advance.

Elisha

**Comment by simonhaenisch** [comment]
Hey I'm not sure what you mean exactly... can you provide some code examples, i. e. what kind of checkmark do you have in your markdown? I'm also not sure what you mean by "converted into text"... there's nothing in the code of md-to-pdf that would do that, at least not that I'm aware of.

**Reply by e2045** [reply]
Hi @simonhaenisch,

sorry for being unclear. 

I have a md file like this
![grafik](https://user-images.githubusercontent.com/51294439/155114726-40eaf17f-1906-42d3-8909-3adfb18c6ddb.png)

After conversion my pdf should be like this
![grafik](https://user-images.githubusercontent.com/51294439/155114855-43ba22ab-2440-4941-b249-8a572f410224.png)

But the result is
![grafik](https://user-images.githubusercontent.com/51294439/155115009-8ab03b81-f1cd-4681-b702-94090b8fc291.png)

My code:

```
function converting(file){
    //converting md to pdf
    (async () => {
        let pdf = await mdToPdf(
            {path: file},
            {pdf_options: {
                printBackground: true,
                format: 'a4',
                margin: {
                    top: '30mm',
                    right: '20mm',
                    bottom: '30mm',
                    left: '20mm',
                },
            }
        }).catch(console.error);

        if (pdf) {
            let newFile = file.substring(0, (file.length - 3)) + ".pdf"
            fs.writeFileSync(newFile, pdf.content);
            console.log(newFile + ' is written');
        }

    })();
}
```

How could I handle that my pdf has check mark emoji after conversion?


My files: 
[checkmark.md](https://github.com/simonhaenisch/md-to-pdf/files/8115820/checkmark.md)
[checkmark.pdf](https://github.com/simonhaenisch/md-to-pdf/files/8115822/checkmark.pdf)

Thank you in advance!

**Reply by e2045** [reply]
Hi @simonhaenisch 

i found a solution from [#107](https://github.com/simonhaenisch/md-to-pdf/discussions/107).

Thank you :D

**Comment by e2045** [comment, marked as answer]
solution in #107
